Mogakco/Sources/Presentation/Profile/ViewController/ProfileViewController.swift

```diff
@@ -220,6 +223,16 @@ final class ProfileViewController: UIViewController {
             .disposed(by: disposeBag)
     }
     
+    private func bindNavigation(output: ProfileViewModel.Output) {
+        rx.viewWillAppear
+            .withLatestFrom(output.navigationBarHidden)
+            .debug()
+            .subscribe(onNext: { [weak self] in
+                self?.navigationController?.setNavigationBarHidden($0, animated: true)
+            })
+            .disposed(by: disposeBag)
+    }
```

Comment:
```suggestion
        output.isMyProfile
            .subscribe(onNext: { [weak self] in
                self?.navigationController?.setNavigationBarHidden($0, animated: true)
            })
            .disposed(by: disposeBag)
    }
```

isMyProfile 정보로 네비게이션 처리하기는 별로일까요?!

Reply:
스터디 상세 -> 프로필로 갈 때는 isMyProfile과는 상관없이 무조건 내비게이션 바를 히든 처리해야 해서 isMyProfile을 사용하지 않았습니다.

Reply:
아아 맞네요! 네비게이션 푸쉬하면서 자신의 프로필이 뜰 수도 있겠네요! 감사합니다:) @sominn9